docs/developer/extending/apps/developing-apps/troubleshooting.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Troubleshooting
+---
+
+Find solutions to common issues that you may face during apps development.
+# Issues
+## I see Typescript errors
+
+Saleor [Apps](https://github.com/saleor/apps), [SDK](https://github.com/saleor/saleor-app-sdk) and [App Template](https://github.com/saleor/saleor-app-template/discussions)
+use [pnpm](https://pnpm.io/) package manager. It can be sensitive with dependencies resolutions, especially in a monorepo setup.
+
+Please ensure you have installed the right version of pnpm on your machine and re-install dependencies with `pnpm install`.
+
+To find the accurate pnpm version in your project, check the root `package.json` file and look for the `packageManager` field.
+
+For example: `"packageManager": "pnpm@8.2.0"` means that you need to install exactly the `8.2.0` version of pnpm.
```

Comment:
That's good info but also something I'd like to see addressed eventually. Requiring precise micro versions of a globally-installed package requires the use of virtual environments, and I feel we need to document that if we can't have it work with a broader set of package manager releases.